group_vars/all

```diff
@@ -66,3 +66,11 @@ calico_installation_type: manifest
 
 ##### List of images to be prepulled #####
 prepull_images: []
+
+##### ETCD version #####
+etcd_version: v3.5.9
+
+##### CNI Configurations #####
+cni_plugins_version: v1.3.0
+cni_plugins_url: https://github.com/containernetworking/plugins/releases/download
+cni_plugins_tarball: "{{ cni_plugins_version }}/cni-plugins-linux-ppc64le-{{ cni_plugins_version }}.tgz"
```

Comment:
Just a thought, going by `cni_plugins_tarball`, I feel it may be better for it to just contain the tar-file name. We can have the `cni_plugin_version` added when generating the URL.

```suggestion
cni_plugins_tarball: "cni-plugins-linux-ppc64le-{{ cni_plugins_version }}.tgz"
```

Reply:
I just followed the convention here https://github.com/kubernetes/kubernetes/blob/master/hack/local-up-cluster.sh#L1168-L1234